java/src/jmri/jmrix/nce/ph5driver/ConnectionConfig.java

```diff
@@ -0,0 +1,46 @@
+package jmri.jmrix.nce.ph5driver;
+
+/**
+ * Definition of objects to handle configuring a layout connection via an NCE
+ * Ph5DriverAdapter object.
+ *
+ * @author Bob Jacobsen Copyright (C) 2001, 2003
+ * @author kcameron Copyright (C) 2010, 2023
+ */
+public class ConnectionConfig extends jmri.jmrix.AbstractSerialConnectionConfig {
+
+    public final static String NAME = Bundle.getMessage("TypePh5USB");
```

Comment:
@KenC57 @bobjacobsen 
I think this string should not be in the bundle. If the string differs between languages, it might cause problem if a user sends his profile to a developer for support. If the developer uses one language and the user another language, and this name differs between these languages, I think the profile will not load due to failure to load this connection.

Reply:
I see your point.  The basic NCE serial connection used a Bundle reference for `TypeSerial` here.  That's defined in jmri.jmrix.Bundle as `Serial` with translations to Dutch, German, and French.  Since `TypePh5USB` is defined as "PH5 USB", maybe that's an arbitrary enough acronym that we can leave it as an explicit String for now.